src/paho/mqtt/client.py

```diff
@@ -353,10 +353,7 @@ def __ne__(self, other):
 
     @property
     def topic(self):
-        if sys.version_info[0] >= 3:
-            return self._topic.decode('utf-8')
-        else:
-            return self._topic
+        return self._topic.decode('utf-8')
```

Comment:
The version check is unnecessary here since `str` in Python 2 accepts `decode` just fine:

```python
Python 2.7.13 |Anaconda 2.5.0 (x86_64)| (default, Dec 20 2016, 23:05:08) 
[GCC 4.2.1 Compatible Apple LLVM 6.0 (clang-600.0.57)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
Anaconda is brought to you by Continuum Analytics.
Please check out: http://continuum.io/thanks and https://anaconda.org
>>> u'some string'.decode('utf-8')
u'some string'
>>> 'some string'.decode('utf-8')
u'some string'
```